ext/hal/nordic/nrfx_config_nrf51.h

```diff
@@ -1008,10 +1008,18 @@
 // <1=> NRF_GPIO_PIN_PULLDOWN
 // <3=> NRF_GPIO_PIN_PULLUP
 
-#ifndef NRFX_SPI_MISO_PULL_CFG
+#ifdef CONFIG_SPI_NRFX_SPI_MISO_NO_PULL
+#define NRFX_SPI_MISO_PULL_CFG 0
+#endif
+
+#ifdef CONFIG_SPI_NRFX_SPI_MISO_PULL_DOWN
 #define NRFX_SPI_MISO_PULL_CFG 1
 #endif
 
+#ifdef CONFIG_SPI_NRFX_SPI_MISO_PULL_UP
+#define NRFX_SPI_MISO_PULL_CFG 2
```

Comment:
This value should be `3`. It is not an index, it must correspond to the value of `GPIO_PIN_CNF_PULL_Pullup` as it is directly written to the configuration register.
Same applies to the other SoCs as well.
Sorry, I missed this before.

Reply:
I am sorry I pushed with this. The documentation is right above and still I mechanically incremented the indexes...
Thank you